dashboard/assets/highcharts_renderer.js

```
@@ -0,0 +1,211 @@
+  if (!obj || typeof obj !== 'object') return obj;
+
+  function revive(value) {
+    if (typeof value === 'string') {
+      var s = value.trim();
+      var looksLikeFn = s.indexOf('function') === 0 || s.indexOf('() =>') === 0 || s.indexOf('(function') === 0;
+      if (looksLikeFn) {
+        try {
+          /* eslint no-new-func: 0 */
+          // Wrap in parentheses to allow function expression parsing
+          // Example: new Function('return ' + s)()
+          return (0, eval)('(' + s + ')');
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)